examples/sdk/sdk_example.py

```diff
@@ -0,0 +1,111 @@
+import sys, os, time
+import aztk.spark
+from aztk.error import AztkError
+
+# set your secrets
+secrets_confg = aztk.spark.models.SecretsConfiguration(
+    batch_account_name="",
+    batch_account_key="",
+    batch_service_url="",
+    storage_account_name="",
+    storage_account_key="",
+    storage_account_suffix="",
+    ssh_pub_key=""
+)
+
+# set path to root of repository to reference files
+ROOT_PATH = os.path.normpath(os.path.join(os.path.dirname(__file__), '..', '..'))
+
+# create a client
+client = aztk.spark.Client(secrets_confg)
+
+# list available clusters
+clusters = client.list_clusters()
+
+# define a custom script
+custom_script = aztk.spark.models.CustomScript(
+    name="simple.sh",
+    script=os.path.join(ROOT_PATH, 'custom-scripts', 'simple.sh'),
+    run_on="all-nodes")
+
+# define spark configuration
+spark_conf = aztk.spark.models.SparkConfiguration(
+    spark_defaults_conf=os.path.join(ROOT_PATH, 'config', 'spark-defaults.conf'),
+    spark_env_sh=os.path.join(ROOT_PATH, 'config', 'spark-env.sh')
+)
+
+# configure my cluster
+cluster_config = aztk.spark.models.ClusterConfiguration(
+    cluster_id="sdk_test",
+    vm_low_pri_count=2,
+    vm_size="standard_f2",
+    custom_scripts=[custom_script],
+    spark_configuration=spark_conf
+)
+
+# create a cluster, and wait until it is ready
+try:
+    cluster = client.create_cluster(cluster_config)
+    cluster = client.wait_until_cluster_is_ready(cluster.id)
+except AztkError as e:
+    print(e.message)
+    sys.exit()
+
+# get details of specific cluster
+cluster = client.get_cluster(cluster_config.cluster_id)
+
+# # create a user for the cluster
+client.create_user(cluster.id, "sdk_example_user", "example_password")
+
+# create some apps to run
```

Comment:
very minor - can we add one app which is scala/java as well?

Reply:
We don't distribute the compiled java/scala programs in the repo. So if I do a java/scala example, the script will either not work out of the box or the java/scala app will need to be commented out. 

Not sure which is more preferable, but I would lean towards leaving it commented out and having the script work by default.